src/Context/token-context.js

```diff
@@ -0,0 +1,21 @@
+import { createContext, useContext, useState } from "react";
+
+const TokenContext = createContext();
+
+const TokenProvider = ({ children }) => {
+  var encodedTokenLocal = localStorage.getItem("ENCODED_TOKEN_2");
+
```

Comment:
again no need to create a token context, create a userContext and there store all the data